sci-libs/calculix-cgx/files/01_cgx_2.8_Makefile_custom_cxx_flags.patch

```diff
@@ -0,0 +1,76 @@
+diff -rupN CalculiX/cgx_2.8/src/Makefile CalculiX.patched/cgx_2.8/src/Makefile
```

Comment:
A little header like described in http://dev.gentoo.org/~vapier/clean-patches would be nice.